src/Ccap/Codegen/Purescript.purs

```diff
@@ -269,21 +270,21 @@ record props = do
 recordJsonCodec :: Array RecordProp -> Codegen Box
 recordJsonCodec props = do
   tell
-    [ { mod: "Data.Argonaut.Core", typ: Nothing, alias: Just "Argonaut" }
-    , { mod: "Ccap.Codegen.Runtime", typ: Nothing, alias: Just "Runtime" }
+    [ { mod: "Data.Argonaut.Core", typ: Nothing, alias: Just "A" }
+    , { mod: "Ccap.Codegen.Runtime", typ: Nothing, alias: Just "R" }
     , { mod: "Foreign.Object", typ: Nothing, alias: Just "FO" }
     , { mod: "Prelude", typ: Nothing, alias: Nothing }
```

Comment:
Is this the last remaining source of conflicting imports? (I think we can deal with it later though.)